FrameworkSystem/Service/DynamicMonitoringHandler.py

```diff
@@ -0,0 +1,50 @@
+"""
+This Service provides functionality to read logging information from the ElasticSearch database
+"""
+
+__RCSID__ = "$Id$"
+
+import types
+from DIRAC import gConfig, gLogger, S_OK, S_ERROR
+from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC.FrameworkSystem.DB.DynamicMonitoringDB import DynamicMonitoringDB
+
+class DynamicMonitoringHandler( RequestHandler ):
+
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    """
+    Handler class initialization
+    """
+
+    DynamicMonitoringHandler.elasticDB = DynamicMonitoringDB()
+
+  types_getLastLog = [ types.StringTypes, types.StringTypes ]
+  def export_getLastLog( self, host, component ):
+    """
+    Retrieves the last logging entry for the given component
+    :param str host: Host where the component is installed
+    :param str component: Name of the component
```

Comment:
I did not say that it is easy, but that it was supposed to be easy with the tests. But looking at the tests, I see that they don't test the output, just the ```res['OK']``` so not very useful.